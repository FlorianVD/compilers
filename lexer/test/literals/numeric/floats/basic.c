1.2 1.23 1.234 0.123456 123.456
