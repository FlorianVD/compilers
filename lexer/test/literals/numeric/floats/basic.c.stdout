1:1 -> 1:4          1.2                 FLOAT_LITERAL       
1:5 -> 1:9          1.23                FLOAT_LITERAL       
1:10 -> 1:15        1.234               FLOAT_LITERAL       
1:16 -> 1:24        0.123456            FLOAT_LITERAL       
1:25 -> 1:32        123.456             FLOAT_LITERAL       
